dvc/exceptions.py

```diff
@@ -328,3 +328,10 @@ def __init__(self, target_infos):
 
 class CollectCacheError(DvcException):
     pass
+
+
+class RemoteNotSpecifiedInTargetRepoError(DvcException):
+    def __init__(self, url):
+        super(RemoteNotSpecifiedInTargetRepoError, self).__init__(
+            "No remote is specified in target repository '{}'".format(url)
```

Comment:
Specify "DVC remote"? Or "remote storage"? Git also has remotes.

Reply:
Good point, Ill include that.

Reply:
Thanks